<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaintingWidget</class>
 <widget class="QWidget" name="PaintingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1404</width>
    <height>945</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Панель рисования</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </item>
   <item>
    <widget class="QTabWidget" name="main_panel_tab_widget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>130</height>
      </size>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>56</red>
           <green>37</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>150</red>
           <green>187</green>
           <blue>124</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Shadow">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>150</red>
           <green>187</green>
           <blue>124</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Highlight">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>18</red>
           <green>59</green>
           <blue>54</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>56</red>
           <green>37</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>150</red>
           <green>187</green>
           <blue>124</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Shadow">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>150</red>
           <green>187</green>
           <blue>124</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Highlight">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>18</red>
           <green>59</green>
           <blue>54</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>150</red>
           <green>187</green>
           <blue>124</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>150</red>
           <green>187</green>
           <blue>124</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Shadow">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>150</red>
           <green>187</green>
           <blue>124</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Highlight">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>120</green>
           <blue>215</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="drawing_tab">
      <attribute name="title">
       <string>Рисовать</string>
      </attribute>
      <widget class="Line" name="enter_data_delimeter_line">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>10</y>
         <width>16</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="save_changes_button">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>940</x>
         <y>20</y>
         <width>80</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>23</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Сохранить&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../Helpers/resoures.qrc">
         <normaloff>:/imgs/Images/save.png</normaloff>:/imgs/Images/save.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="discard_changes_button">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>1030</x>
         <y>20</y>
         <width>80</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>23</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Отмена&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../Helpers/resoures.qrc">
         <normaloff>:/imgs/Images/cansel.png</normaloff>:/imgs/Images/cansel.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
      <widget class="Line" name="save_changes_delimeter_line">
       <property name="geometry">
        <rect>
         <x>920</x>
         <y>10</y>
         <width>16</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="return_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>31</width>
         <height>25</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Отменить&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../Helpers/resoures.qrc">
         <normaloff>:/imgs/Images/arrow_1.png</normaloff>:/imgs/Images/arrow_1.png</iconset>
       </property>
      </widget>
      <widget class="QPushButton" name="forward_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>31</width>
         <height>25</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Повторить действие&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../Helpers/resoures.qrc">
         <normaloff>:/imgs/Images/arrow_2.png</normaloff>:/imgs/Images/arrow_2.png</iconset>
       </property>
      </widget>
      <widget class="QPushButton" name="enable_grid_button">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>800</x>
         <y>20</y>
         <width>121</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Включить сетку</string>
       </property>
      </widget>
      <widget class="Line" name="brush_setting_delimeter_line">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>10</y>
         <width>16</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QLabel" name="brush_label">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>10</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Кисть:</string>
       </property>
      </widget>
      <widget class="QSlider" name="thickness_horizontal_slider">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>70</y>
         <width>201</width>
         <height>18</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="thickness_label">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>40</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Толщина:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="brush_type_1_button">
       <property name="geometry">
        <rect>
         <x>258</x>
         <y>10</y>
         <width>31</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../arrow_2.png</normaloff>../../../arrow_2.png</iconset>
       </property>
      </widget>
      <widget class="QPushButton" name="brush_type_2_button">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>10</y>
         <width>31</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>2</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../arrow_2.png</normaloff>../../../arrow_2.png</iconset>
       </property>
      </widget>
      <widget class="QPushButton" name="brush_type_3_button">
       <property name="geometry">
        <rect>
         <x>342</x>
         <y>10</y>
         <width>31</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>3</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../arrow_2.png</normaloff>../../../arrow_2.png</iconset>
       </property>
      </widget>
      <widget class="QPushButton" name="brush_type_4_button">
       <property name="geometry">
        <rect>
         <x>384</x>
         <y>10</y>
         <width>31</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>4</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../arrow_2.png</normaloff>../../../arrow_2.png</iconset>
       </property>
      </widget>
      <widget class="Line" name="display_delimeter_line">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>10</y>
         <width>16</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QLabel" name="border_label">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>10</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Граница:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="shadow_nonWO_button">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>800</x>
         <y>60</y>
         <width>121</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Затемнить</string>
       </property>
      </widget>
      <widget class="QLabel" name="grid_display_label">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>20</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Отображение:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="clear_content_button">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>71</width>
         <height>60</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Перерисовать область на изображении&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Перерисовать\nобласть на\nизображении&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../Helpers/resoures.qrc">
         <normaloff>:/imgs/Images/pncl.png</normaloff>:/imgs/Images/pncl.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="change_grid_parameters_button">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>30</y>
         <width>100</width>
         <height>60</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>60</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Изменить
параметры сетки</string>
       </property>
      </widget>
      <widget class="QPushButton" name="create_grid_button">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>60</y>
         <width>141</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Построить сетку</string>
       </property>
      </widget>
      <widget class="Line" name="current_source_delimeter_line">
       <property name="geometry">
        <rect>
         <x>1120</x>
         <y>20</y>
         <width>16</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QGroupBox" name="sources_group_box">
       <property name="geometry">
        <rect>
         <x>1140</x>
         <y>9</y>
         <width>131</width>
         <height>81</height>
        </rect>
       </property>
       <property name="title">
        <string>Источники</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QComboBox" name="source_name_combo_box"/>
        </item>
        <item>
         <widget class="QComboBox" name="source_type_combo_box">
          <property name="editable">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>Точечный</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Диффузионный</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGraphicsView" name="map_editor_graphics_view">
     <property name="minimumSize">
      <size>
       <width>1280</width>
       <height>720</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Helpers/resoures.qrc"/>
 </resources>
 <connections/>
</ui>
